dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -4985,6 +5002,35 @@ public Node visitOther(Tree tree, Void p) {
         }
     }
 
+    /**
+     * Determine whether loop condition {@link JCTree} is a constant true, according to the compiler
```

Comment:
This method works for any tree, even if that tree is not a loop condition.  We might find uses for this method outside of this class, would you please move it to `TreeUtils` and rename it to `isLiteralTrue`.

Reply:
Moving to `TreeUtils` sounds good. As name I would use `isExpressionConstantTrue` or `isExprConstTrue`, or the current `isCondConstTrue`.

Reply:
The Javadoc should say `is a compile-time constant true` or as the JLS uses `a condition expression has the constant value true`.  (I clearly misunderstood the term `constant true`, other might, too.)